pywbem_mock/_defaultinstanceprovider.py

```diff
@@ -821,8 +793,16 @@ class MethodProvider(BaseProvider):
 
     User providers are defined by creating a subclass of this class and
     defining an InvokeMethod based on the method in this class.
+
+    Attributes:
+
+      provider_type (:term:`string`):
+        Keyword defining the type of request the provider will service.
+        The type for this class is 'method'
     """
 
+    provider_type = 'method'
+
     def __init__(self, cimrepository=None):
         """
         Parameters:
```

Comment:
On the `InvokeMethod` method definition:
1. Its list of parameters is mostly in lower case but should be mostly in mixed case, consistent with WBEMConnection.InvokeMethod. Its docstring has it correct.
2. Its docstring misses an entry for the `namespace` parameter.

Reply:
Added namespace, Changed parameters here and in tests to Mixed case per docstring. DONE

Reply:
DONE.